app/assets/javascripts/alchemy/alchemy.link_dialog.js.coffee

```diff
@@ -48,64 +66,10 @@ class window.Alchemy.LinkDialog extends Alchemy.Dialog
         target: $("##{@link_type}_link_target").val()
       false
 
-  # Initializes the select2 based Page select
-  initPageSelect: ->
```

Comment:
This should maybe be its separate PR that builds on top of the remaining commits?